src/entity/humanoid.rs

```diff
@@ -23,25 +28,51 @@ const PLAYER_LEG_LENGTH: f32 = PLAYER_BODY_LENGTH;
 const PLAYER_ARM_LENGTH: f32 = PLAYER_BODY_LENGTH;
 
 const PLAYER_LEG_WIDTH_SHIFT: f32 = (PLAYER_BODY_WIDTH - PLAYER_LEG_WIDTH) / 2.;
-const PLAYER_ARM_WIDTH_SHIFT: f32 = 1.01 * (PLAYER_BODY_WIDTH + PLAYER_ARM_WIDTH ) / 2.;
+const PLAYER_ARM_WIDTH_SHIFT: f32 = 1.01 * (PLAYER_BODY_WIDTH + PLAYER_ARM_WIDTH) / 2.;
 
 const PLAYER_BODY_SCALE: [f32; 3] = [PLAYER_BODY_LENGTH, PLAYER_BODY_HEIGHT, PLAYER_BODY_WIDTH];
 const PLAYER_ARM_SCALE: [f32; 3] = [PLAYER_ARM_LENGTH, PLAYER_ARM_HEIGHT, PLAYER_ARM_WIDTH];
 const PLAYER_LEG_SCALE: [f32; 3] = [PLAYER_LEG_LENGTH, PLAYER_LEG_HEIGHT, PLAYER_LEG_WIDTH];
 
-// Define the position of the textures to use for the player
-// TODO if this remain public, it needs a much clearer documentation
-pub const PATRON_PLAYER_CUT: [[u32; 4]; 24] = [
-    // Head
-    [0, 24, 8, 8], [8, 24, 8, 8], [16, 24, 8, 8], [24, 24, 8, 8], [32, 24, 8, 8], [40, 24, 8, 8], 
-    // Leg
-    [0, 12, 4, 12], [4, 12, 4, 12], [8, 12, 4, 12], [12, 12, 4, 12], [16, 20, 4, 4], [20, 20, 4, 4],
-    // Body
-    [0, 0, 4, 12], [4, 0, 8, 12], [20, 0, 4, 12], [12, 0, 8, 12], [24, 0, 8, 4], [32, 0, 8, 4], 
-    // Arm
-    [24, 12, 4, 12], [28, 12, 4, 12], [32, 12, 4, 12], [36, 12, 4, 12], [28, 8, 4, 4], [32, 8, 4, 4],
+// Define how to cut the image of the player to generate the textures for the player
+// Values are in (u,v) coord, in fraction of the image dimension
+const PATRON_PLAYER_CUT: [ImageCut; 24] = [
```

Comment:
`PATRON` is a french world.

Maybe use `CUT_TEMPLATE`  or `PLAYER_CUT_TEMPLATE`